### PR TITLE
[Hackathon117] Add encrypted personalization data using ES-ECDH 

### PR DESCRIPTION
**Pull request by kentakayama**
Solves #330 
NOTE: I've generated this binary using t_cose 2 alpha but its status is still fixing some bugs.
Please wait until I can validate this.

### CODE REVIEW

**Comment by kentakayama** [comment]
- :heavy_check_mark:  This example is validated by processing with the latest [libcsuit](https://github.com/kentakayama/libcsuit/pull/28).
- :warning: Need to test with other implementation to decrypt the COSE_Encrypt.

**Comment by hannestschofenig** [comment]
I think an example would be very useful!

**Comment by kentakayama** [comment]
This example has updated to use KDF context and now ready to merge.

File: draft-ietf-teep-protocol.md [hunk 1]
```diff
@@ -2126,6 +2126,13 @@ bz/m4rVlnIXbwK07HypLbAmBMcCjbazR14vTgdzfsJwFLbM5kdtzOLSolg==
 ## Example 3: Supplying Personalization Data for Trusted Component Binary {#suit-personalization}
 {: numbered='no'}
 
+This example uses the following parameters:
+
+- Algorithm for payload encryption: AES-GCM-128
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:2131]: I think this is obsolete now based on #356 and #373?

Comment (hannestschofenig) [reply draft-ietf-teep-protocol.md:2131]: As noted in my comments to the issue, I think that the use of AES-GCM is correct.

**Review by hannestschofenig — APPROVED** [review]
The change looks good to me.

File: draft-ietf-teep-protocol.md (hunk 1 above)
Comment (dthaler) [reply draft-ietf-teep-protocol.md:2131]: Sent email to the list to discuss, since AES-GCM is not in suit-mti.

**Comment by mingpeiwk** [comment]
Looks good for me too.

**Review by mingpeiwk — APPROVED** [review]

**Comment by hannestschofenig** [comment]
Can we merge this PR?

**Comment by dthaler** [comment]
In my view this should not be merged until suit-mti changes.

**Review by dthaler — APPROVED** [review]

**Pull request merged**